help/destinations/catalog/voice/medallia-connector.md

```diff
@@ -0,0 +1,108 @@
+
+
+### Fill in destination details {#destination-details}
+To configure details for the destination, fill in the required fields and select **[!UICONTROL Next]**.
+
+*  **[!UICONTROL Name]**: A name by which you will recognize this destination in the future.
+*  **[!UICONTROL Description]**: A description that will help you identify this destination in the future.
+*  **[!UICONTROL Import API Name]**: Name of the Medallia Import API (also known as Web Feed) to be used in this connection. Youcan activate different segments to different Import APIs to trigger different survey programs
+
+<img src="/help/destinations/assets/catalog/voice/medallia-destination-details.png" width="60%">
+
+
+## Activate segments to this destination {#activate}
+
+>[!IMPORTANT]
+> 
+>To activate data, you need the **[!UICONTROL Manage Destinations]**, **[!UICONTROL Activate Destinations]**, **[!UICONTROL View Profiles]**, and **[!UICONTROL View Segments]** [access control permissions](/help/access-control/home.md#permissions). Read the [access control overview](/help/access-control/ui/overview.md) or contact your product administrator to obtain the required permissions.
+
+Read [Activate profiles and segments to streaming segment export destinations](../../ui/activate/activate-segment-streaming-destinations.md) for instructions on activating audience segments to this destination.
+
+### Map attributes and identities {#map}
+The following target identity namespace(s) must be mapped depending on the use case:
+* For email-based surveys, **email** must be mapped as a target field using **Target field** > **Select identity namespace** > **email**
+* For SMS-based surveys, **phone** must be mapped as a target field using **Target field** > **Select identity namespace** > **phone**. Phone numbers must be in E.164 format, which includes a plus sign (+), an international country calling code, a local area code, and a phone number
+
+It is strongly recommended that you also map additional target custom attributes to create personalized surveys and append additional information about the customer to the survey record:
+* Personalized surveys typically address the customer by name
+    *  Map the customer's first name to **Target field** > **Select custom attributes** > **Attribute name** > **firstname**
+    *  Map the customer's last name to **Target field** > **Select custom attributes** > **Attribute name** > **lastname**
+* Add mappings for any other target custom attributes as desired
+
+![Image of sample mapping](/help/destinations/assets/catalog/voice/medallia-destination-mapping.png)
+
+
+>[!IMPORTANT]
+> 
+> Share with your Medallia delivery team the exact **Attribute names** for every target custom attribute you map using **Target field** > **Select custom attributes** > **Attribute name**. You may wish to take a screenshot of the mapping page to share directly.
+
+
+## Exported data {#exported-data}
```

Comment:
One comment here. The purpose of this section is for readers to verify that the data is indeed being correctly exported to your destination. 

So, for example, you could tell them to go into the Medallia interface and check that the profiles or segments out of Experience Platform are being populated.